### DISCUSSION TITLE
Any plans to translate Mojo Docs into Kor?

### DISCUSSION THREAD

**Discussion by airplane2230** [modular/modular#421] [Q&A] [answered]
I am very interested in Mojo. 
Do you have any plans to translate Mojo Docs into other languages? 

I want to participate or proceed with a project that translates from English to Korean.

**Comment by lattner** [comment, marked as answer]
We'd love help with translations some day, but the documentation is still pretty immature.  @scottamain has some plans to upgrade the current manual (e.g. make it more "normal programmer" focused instead of "language nerd" focused :-) )  and I think it would make sense to wait for that.

**Comment by jay-m-dev** [comment]
If using Google Chrome, you can select to translate the page by right clicking next to the tab name in the playground, then select **translate to [language]**. You may need to install the Google Translate extension for this. The translations aren't perfect, but have improved over time, this may be a good workaround until formal translations are implemented.

![image](https://github.com/modularml/mojo/assets/20008149/bc66f76f-70e2-4efb-b36e-493db24148cb)
